scss/mediaQueries.scss

```diff
@@ -0,0 +1,202 @@
+@media (min-width: 620px) {
```

Comment:
Grouping all MQ's in one file might work for a small project, but it's not scalable. Quickly you can have a file with 1000's of lines. A better approach is to place all MQ's at the bottom of the file they refer to.

Reply:
Totally agree, I won't make such a mistake in the future :D